### DISCUSSION TITLE
Accept an Inline Edit

### DISCUSSION THREAD

**Discussion by maxpetretta** [olimorris/codecompanion.nvim#77] [Q&A] [answered]
maybe a silly question, so forgive me: how can i "accept" an inline edit, such that the deleted lines are cleared from the buffer?

so far only closing/reopening neovim removes these lines

![CleanShot 2024-07-26 at 21 09 06@2x](https://github.com/user-attachments/assets/b6f715f3-6955-441a-8958-219c0fe05763)

**Comment by maxpetretta** [comment, marked as answer]
🤦‍♂️ almost immediately after posting, found it in the default config:
https://github.com/olimorris/codecompanion.nvim/blob/ca79c0475abf7fc20dfb95c6238e2a24e1f758c1/lua/codecompanion/config.lua#L57

**Reply by olimorris** [reply]
This should definitely be in the README. Thanks for raising

**Comment by olimorris** [comment]
@maxpetretta as per a78c4b59009af197a9d783d657c0b20330bf8c4a, I've now added some keymaps to allow you to accept or reject an edit. `ga` to accept or `gr` to reject. Both of them are customisable in the config

**Reply by maxpetretta** [reply]
amazing, ty!
